src/OAuthClient/BibClient.php

```diff
@@ -9,13 +9,27 @@
 class BibClient extends APIClient
 {
     /**
-     * @param string $bibId
+     * @param array $bibIds
      * @param $nyplSource
-     * @return null|Bib
+     * @return array|null
      */
-    public static function getBibByIdAndSource($bibId = '', $nyplSource)
+    public static function getBibsByIds(array $bibIds)
     {
-        $url = Config::get('API_BIB_URL') . '/' . $nyplSource . '/' . $bibId;
+        if (count($bibIds) < 1) {
+            APILogger::addError(
+                'Failed',
+                array('No bibIds provided to ' . __FUNCTION__)
+            );
+            return null;
+        } elseif (count($bibIds) > 1) {
```

Comment:
I know you might be refactoring out the `elseif` but just as a heads-up, usually an `else` or, even worse, an `elseif` is a code smell that the logic can be refactored into a separate method. For example, in this case, it could be `getBibStringFromArray` or whatever you call it. I think it makes the code cleaner, more self-documenting, and methods smaller.